### PR TITLE
docs: add Code of Conduct

### PR DESCRIPTION
**Pull request by ephys**
### Description Of Change

This PR adds a Code of Conduct for the organization.

It's a bit of a mash up of 4 different code of conducts

Let me know if there are parts you disagree with, or if there are things you'd like to add

I've also updated `CONTACT.md` to add my email and specify which maintainers are ready to accept reports for CoC violations and security issues.
I've only added that to my own email right now because I don't want to assume who is willing to handle these.
You can accept one, both, or none. Let me know and I'll update the file :)

### CODE REVIEW

**Review by sdepold — DISMISSED** [review]
good stuff :)

**Review by WikiRik — COMMENTED** [review]
Thanks for making this PR! It's good to have a CoC. I just have one remark; it's now focused on events but I think this should cover the GitHub repos and Slack as well. Not sure what would be a good wording for that though

**Comment by ephys** [comment]
Can you see if the update I just pushed resolves that issue? It's true that part of the copy I included was too specific to events, instead of the day to day that happens on GitHub and slack

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Code of Conduct
+
+All participants of Sequelize are expected to abide by our Code of Conduct,
+both online (be it on GitHub, slack, and others) and during in-person events (such as meetups) that are hosted and/or associated with Sequelize.
```
Comment (WikiRik) [inline CODE-OF-CONDUCT.md:4]:
```suggestion
both online (be it on GitHub, Slack, and others) and during in-person events (such as meetups) that are hosted and/or associated with Sequelize.
```

**Review by WikiRik — DISMISSED** [review]
Yes, this is better. Thanks!

**Review by WikiRik — APPROVED** [review]

**Pull request merged**